src/positionable/IsometricPositionable.cpp

```diff
@@ -27,6 +26,9 @@ void IsometricPositionable::_register_methods() {
     register_property("size3d", &IsometricPositionable::setSize3D, &IsometricPositionable::getSize3D, Vector3(1, 1, 1));
     register_property("is_temporary", &IsometricPositionable::setTemporary, &IsometricPositionable::isTemporary, true);
     register_property("debug_z", &IsometricPositionable::setDebugZ, &IsometricPositionable::getDebugZ, 0);
+    register_property("slope_type", &IsometricPositionable::setSlopeType, &IsometricPositionable::getSlopeType,

```

Comment:
Adding the slope here means that a map can be a slope as well. Not very logical

Reply:
Yep, that's true.
I made a new commit to make it virtual so that IsoMap can only be none.